GA/utils.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+class Utils:
+    # Dictionary mapping slot numbers to time intervals in 12-hour format
+    time_slots = {
```

Comment:
Try importing from the samples. All these will be imported from a standard place.